inc/Engine/Media/LazyloadSubscriber.php

```diff
@@ -138,36 +109,69 @@ public function insert_lazyload_script() {
 		 *
 		 * @param bool $polyfill True to use the polyfill, false otherwise.
 		 */
-		$polyfill = apply_filters( 'rocket_lazyload_polyfill', false );
+		$polyfill = (bool) apply_filters( 'rocket_lazyload_polyfill', false );
 
 		$script_args = [
-			'base_url' => WP_ROCKET_ASSETS_JS_URL . 'lazyload/',
-			'version'  => '12.0',
+			'base_url' => rocket_get_constant( 'WP_ROCKET_ASSETS_JS_URL' ) . 'lazyload/',
+			'version'  => self::SCRIPT_VERSION,
 			'polyfill' => $polyfill,
 		];
 
+		$inline_script = $this->assets->getInlineLazyloadScript( $this->set_inline_script_args() );
+
+		if ( ! rocket_get_constant( 'SCRIPT_DEBUG' ) ) {
+			$minify = new JS( $inline_script );
+
+			$inline_script = $minify->minify();
+		}
```

Comment:
- The `$inline_script` is conditional: it'll be set by either assets or the minify. Let's make it conditional. Why? It's faster as we shouldn't invoke `assets->getInlineLazyloadScript()` unless the constant is set. One or the other.

- Also the `new JS` here makes it harder to test. We can't mock it. 

- Let's move all of this code to a separate method and let that method get the inline script.